db/schema.rb

```diff
@@ -1350,6 +1350,7 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
     t.index ["twitter_username"], name: "index_users_on_twitter_username", unique: true
     t.index ["username"], name: "index_users_on_username", unique: true
+    t.check_constraint "username IS NOT NULL", name: "users_username_not_null"
```

Comment:
Rails 6.1 finally dumps check constraints, see https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html#method-i-check_constraint

@citizen428 will be happy :D